h3.go

```diff
@@ -104,44 +142,44 @@ func NewLatLng(lat, lng float64) LatLng {
 }
 
 // LatLngToCell returns the Cell at resolution for a geographic coordinate.
-func LatLngToCell(latLng LatLng, resolution int) Cell {
+func LatLngToCell(latLng LatLng, resolution int) (Cell, error) {
 	var i C.H3Index
 
-	C.latLngToCell(latLng.toCPtr(), C.int(resolution), &i)
+	errC := C.latLngToCell(latLng.toCPtr(), C.int(resolution), &i)
 
-	return Cell(i)
+	return Cell(i), errMap[errC]
```

Comment:
This happens a lot in the code right, but there's one interesting behavior w/ a map here. I think could be a risk here when not checking for the key in the map since the zero value of an error is `nil` and so if an error code is introduced, this would return `nil`.

Reply:
I also thought about this, do you think H3 core would return other error codes?

Reply:
Done

Reply:
@mojixcoder Thank you!

To your question, they've (h3 core) certainly given themselves space to add error codes w/ the uint here and I'm sure this set grew over time as well. Just a little future-proofing on our end.